### PR TITLE
Documentation for `name` query parameter

### PR DESCRIPTION
**Pull request by MobiusHorizons**
Adding docs based on https://github.com/webpack-contrib/bundle-loader/issues/31

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,13 +45,38 @@ load(function(file) {
...
 
-You may set name for bundle (`name` query parameter). See [documentation](https://github.com/webpack/loader-utils#interpolatename).
+You may set name for a bundle using the `name` query parameter. 
+See [documentation](https://github.com/webpack/loader-utils#interpolatename).
+
+**Note** chunks created by the loader will be named according to the 
+[`output.chunkFilename`](https://webpack.js.org/configuration/output/#output-chunkfilename) rule, which defaults to `[id].[name]`.
+Here `[name]` corresponds to the chunk name set in the `name` query parameter. 
+
+####**Example:**
```
Comment (michael-ciniawsky) [inline README.md]: `#### **Example** => #### Example` No need for the ** here :)

File: README.md [hunk 2]
```diff
@@ -45,13 +45,38 @@ load(function(file) {
...
 
 ``` javascript
```
Comment (michael-ciniawsky) [inline README.md]: ` ``` javascript => ```js `

**Comment by michael-ciniawsky** [comment]
fixes #31

**Comment by MobiusHorizons** [comment]
@michael-ciniawsky updated

**Comment by michael-ciniawsky** [comment]
@MobiusHorizons Thx

**Comment by MobiusHorizons** [comment]
glad to help. Its a very useful plugin!!

**Pull request merged**